docs/book/src/machine-images/vpc.md

```diff
@@ -1 +1,62 @@
-This page is still being written - stay tuned!
```

Comment:
This location is present to host the prebuilt images for the users to use, I'm wondering if we can move this page under [Developer Guide](https://deploy-preview-613--kubernetes-sigs-cluster-api-ibmcloud.netlify.app/developer/index.html) section and refer in this page instead.

Reply:
@mkumatag Hi, how about moving my changes from vpc.md to this
https://cluster-api-ibmcloud.sigs.k8s.io/topics/vpc/prerequisites.html
prerequisites part like it is done in

https://cluster-api-ibmcloud.sigs.k8s.io/topics/powervs/prerequisites.html#build-workload-cluster-image

I would do that instead and leave this note in vpc.md
> This page is still being written - stay tuned!

Reply:
I see its already here in the vpc section - https://cluster-api-ibmcloud.sigs.k8s.io/topics/vpc/prerequisites.html#build-workload-cluster-image, I prefer creating a separate page for the building the image itself and refer wherever you want.

Reply:
@mkumatag I pushed my changes to https://github.com/FischerLGLN/cluster-api-provider-ibmcloud :) 
However, I don't see an update in this PR.

Reply:
Ah, now it is there.

Reply:
somehow it shows me when I click the link, something is broken!

<img width="1045" alt="image" src="https://user-images.githubusercontent.com/12646029/162729022-f54a2c5f-b328-4c9b-a434-22a621e044fe.png">

Reply:
@mkumatag 
There is some confusion around markdown and html in this repo.

See here:
https://cluster-api-ibmcloud.sigs.k8s.io/machine-images/vpc.html
https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/docs/book/src/machine-images/vpc.md

I made a reference to a uploading-an-image.md (instead of your opened html) inside the prerequisites.md.
Does this cause this error?

Reply:
@mkumatag According to your screenshot, there should be a 4.1.3 in the left.
Maybe the netlify bot hasn't redeployed my changes.

Reply:
Okay, I can locally test with.
```make serve```
Same bad result, have to investigate :thinking:

Reply:
/test ?